lib/http.go

```diff
@@ -37,7 +37,24 @@ func (c *APIClient) Post(endpoint string, payload interface{}) (*http.Response,
 	}
 
 	req.Header.Set("Content-Type", "application/json")
-	req.Header.Set("Authorization", fmt.Sprintf("Bearer %s", c.APIKey))
+	req.Header.Set("x-api-key", c.APIKey)
```

Comment:
I am not sure if this is the X in x-api-key is capital in the documentation. If it doest affect anything then just ignore it.

Reply:
thaks for this .. the header according to the docs is `x-api-key` for authorization